ironfish/src/blockchain/blockchain.old.test.ts

```diff
@@ -35,39 +35,12 @@ describe('Note adding', () => {
   it('immediately adds in order notes to the tree', async () => {
     await blockchain.addNote(0, 'zero')
     await blockchain.addNote(1, 'one')
-    expect(blockchain.looseNotes[0]).toBe('zero')
-    expect(blockchain.looseNotes[1]).toBe('one')
     expect(await blockchain.notes.size()).toBe(2)
     expect(await blockchain.notes.get(0)).toBe('zero')
     expect(await blockchain.notes.get(1)).toBe('one')
     expect(await blockchain.nullifiers.size()).toBe(0)
     expect(listener).not.toBeCalled()
   })
-  it('adds an out of order note only to the loose notes', async () => {
-    await blockchain.addNote(10, 'ten')
-    await blockchain.addNote(11, 'eleven')
-    await blockchain.addNote(12, 'twelve')
-    expect(blockchain.looseNotes[10]).toBe('ten')
-    expect(blockchain.looseNotes[11]).toBe('eleven')
-    expect(blockchain.looseNotes[12]).toBe('twelve')
-    expect(await blockchain.notes.size()).toBe(0)
-    expect(await blockchain.nullifiers.size()).toBe(0)
-    expect(listener).not.toBeCalled()
-  })
-  it('syncs loose notes to the tree when the gap fills in', async () => {
-    await blockchain.addNote(2, 'two')
-    await blockchain.addNote(1, 'one')
-    await blockchain.addNote(0, 'zero')
-    expect(blockchain.looseNotes[0]).toBe('zero')
-    expect(blockchain.looseNotes[1]).toBe('one')
-    expect(blockchain.looseNotes[2]).toBe('two')
-    expect(await blockchain.notes.size()).toBe(3)
-    expect(await blockchain.notes.get(0)).toBe('zero')
-    expect(await blockchain.notes.get(1)).toBe('one')
-    expect(await blockchain.notes.get(2)).toBe('two')
-    expect(await blockchain.nullifiers.size()).toBe(0)
-    expect(listener).not.toBeCalled()
-  })
   it("logs errors if the note doesn't match the previously inserted note that position", async () => {
     const errorFn = jest.fn()
     blockchain['logger'].mockTypes((type) => {
```

Comment:
This should be deletable now if we don't care to test whether the logger is called